keel-core/src/main/kotlin/com/netflix/spinnaker/config/LifecycleConfig.kt

```diff
@@ -0,0 +1,13 @@
+package com.netflix.spinnaker.config
+
+import org.springframework.boot.context.properties.ConfigurationProperties
+import java.time.Duration
+
```

Comment:
I've set the config here to have 30s between checks of the same task. Is that too long? Too short?

Reply:
Can you elaborate on what is the purpose of this class? (and maybe add docs)

Reply:
@lorin it might be possible, but to be honest I have not seen things quickly transition from baking -> deploying. i think it takes the clouddriver cache > 30s to update. so, I don't think it'll be a problem.

@gal-yardeni this is a config file - what kind of information would you be looking for? an explanation of how the config options will be used?